velox/docs/develop/types.rst

```diff
@@ -143,4 +144,4 @@ The high 52 bits of bigint store signed integer for milliseconds in UTC.
 Supported range of milliseconds is [0xFFF8000000000000L, 0x7FFFFFFFFFFFF]
 (or [-69387-04-22T03:45:14.752, 73326-09-11T20:14:45.247]). The low 12 bits
 store timezone ID. Supported range of timezone ID is [1, 1680].
-The definition of timezone IDs can be found in ``TimeZoneDatabase.cpp``.
\ No newline at end of file
+The definition of timezone IDs can be found in ``TimeZoneDatabase.cpp``.
```

Comment:
Related change?

Reply:
There was no newline at the end of the file. My editor auto-added that.